Formula/haproxy.rb

```diff
@@ -1,8 +1,8 @@
 class Haproxy < Formula
   desc "Reliable, high performance TCP/HTTP load balancer"
   homepage "https://www.haproxy.org/"
-  url "https://www.haproxy.org/download/2.3/src/haproxy-2.3.5.tar.gz"
-  sha256 "7924539530bbf555829c7f5886be0b7fcf8d9c8ffe0867b7010beb670abfbe4b"
+  url "https://www.haproxy.org/download/2.3/src/haproxy-2.3.6.tar.gz"
+  sha256 "6d4620e5da1d93ed75f229011216db81d5097b68bf175e309f2fab2890bba036"
```

Comment:
```suggestion
  sha256 "6d4620e5da1d93ed75f229011216db81d5097b68bf175e309f2fab2890bba036"
  license "GPL-2.0-or-later" => { with: "openvpn-openssl-exception" }
```
From `LICENSE`:
```
Due to the incompatibility between the GPL and the OpenSSL licence, you must
apply the GPL/LGPL licence with the following exception:
This program is released under the GPL with the additional exemption that
compiling, linking, and/or using OpenSSL is allowed.
```

From `src/haproxy.c`:
```
 * This program is free software; you can redistribute it and/or
 * modify it under the terms of the GNU General Public License
 * as published by the Free Software Foundation; either version
 * 2 of the License, or (at your option) any later version.
```